content/guides/getting-started/installing-cypress.md

```diff
@@ -12,57 +12,8 @@ title: Installing Cypress
 
 </Alert>
 
-## System requirements
-
-#### Operating System
-
-Cypress is a desktop application that is installed on your computer. The desktop
-application supports these operating systems:
-
-- **macOS** 10.9 and above _(64-bit only)_
-- **Linux** Ubuntu 12.04 and above, Fedora 21 and Debian 8 _(64-bit only)_
-- **Windows** 7 and above _(64-bit only)_
-
-#### Node.js
-
-If you're using `npm` to install Cypress, we support:
-
-- **Node.js** 12 or 14 and above
-
-#### Linux
-
-If you're using Linux, you'll want to have the required dependencies installed
-on your system.
-
-#### Ubuntu/Debian
-
-```shell
-apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
-```
-
-#### CentOS
-
-```shell
-yum install -y xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib
-```
-
-#### Docker
-
-Docker images with all of the required dependencies installed are available
-under [cypress/base](https://github.com/cypress-io/cypress-docker-images)
-
-If you're running your projects in containers, then you'll want Cypress in the
-container with the Node.js process.
-
-```
-  ui:
-    image: cypress/base:latest
-    # if targeting a specific node version, use e.g.
-    # image: cypress/base:14
-```
-
-`cypress/base` is a drop-in replacement for
-[base docker node images](https://hub.docker.com/_/node/).
+First, make sure you have all the [system requirements](#System-requirements),
+and then...
```

Comment:
```suggestion
First, make sure you have all the [system requirements](#System-requirements).
```

Reply:
This make it back in someone or did you want to keep it?

Reply:
Oops l undid some changes from earlier. This is what happens when you poke at PRs between stuff at a conf 😀 I’ll pull it in again.

Reply:
Oops l undid some changes from earlier. This is what happens when you poke at PRs between stuff at a conf 😀 I’ll pull it in again.